views/partials/header/header.hbs

```diff
@@ -1,7 +1,8 @@
     {{> vpn-banner }}
     <header id="header" class="{{ addUpgradeClasses }}">
       {{{ recruitmentBanner }}}
-      {{{ microsurveyBanner }}}
+      {{!-- {{{ microsurveyBanner }}} --}}
```

Comment:
IIRC we're replacing ALL the micro surveys with this single CSAT? We can probably just remove this line and the corresponding microsurvey logic completely?

Reply:
yes.  Since microsurvey has nothing to do with the current pilot, this would be a good opportunity to de-clutter.  I'll add in a separate PR (possibly along with the signup code above)